lib/hubspot/connection.rb

```diff
@@ -7,7 +7,8 @@ def get_json(path, opts)
         url = generate_url(path, opts)
         response = get(url, format: :json)
         log_request_and_response url, response
-        handle_response(response)
+        raise(Hubspot::RequestError.new(response)) unless response.success?
+        response.parsed_response
```

Comment:
To keep these changes isolated, `get_json` still returns a hash as we focus on making updates consistent.